public/specifications/monitoring/MONITORING.md

```diff
@@ -18,6 +18,19 @@ Pillarbox integrates with a monitoring platform that provides real-time and hist
 
 This article describes the flexible event model that allows our players to send data to our monitoring platform.
 
+## JSON Key / Value Naming Conventions
+
+The following naming conventions are applied for key / values appearing in JSON payloads:
+
+- Keys use snake case.
+- Values are provided in a form best suited for display, usually capitalized, with possible exceptions when brand
+  names are involved (e.g. macOS must not be capitalized as MacOS).
+- The only values provided in uppercase are event types (e.g. `START`).
+
+> [!IMPORTANT]
+> Values provided by the system (e.g. device names) can be assumed as having a form already best suited for display.
+Their case must not be changed.
```

Comment:
```suggestion
> Their case must not be changed.
```

Reply:
Seems not strictly necessary but likely better, thanks.